static_report/package.json

```diff
@@ -22,15 +23,41 @@
     "web-vitals": "^2.1.4",
     "wouter": "2.8.0-alpha.2"
   },
+  "devDependencies": {
+    "@babel/core": "^7.18.2",
+    "@types/d3": "^7.4.0",
+    "@types/jest": "^28.1.2",
+    "@types/node": "^18.0.0",
+    "@types/react": "^18.0.14",
+    "@types/react-dom": "^18.0.5",
+    "chalk": "^5.0.1",
+    "eslint": "^8.17.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "generate-schema": "^2.6.0",
+    "json-schema-to-typescript": "^10.1.5",
+    "node-html-parser": "^5.3.3",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.6.2",
+    "rimraf": "^3.0.2",
+    "typescript": "^4.7.4"
+  },
   "scripts": {
-    "start": "REACT_APP_SINGLE_REPORT=true react-scripts start",
-    "start:compare": "PORT=3001 REACT_APP_SINGLE_REPORT=false react-scripts start",
+    "prestart:single": "run-s schema:* typing:* embed:html",
```

Comment:
`prestart:single` and `prestart:comparison' behavior is the same. Just remain and need to update README.

Reply:
Hmm, the duplication is there because each `prestart:<name>` npm-script needs a matching `start:<name>` to fire. 
Not sure if there's a syntax glob pattern that npm uses.
In any case, I will use `run-s` instead for both `start:<name>` scripts, that way we don't need two duplicate prestart scripts

Reply:
I like using `npm run setup` *(Actually, setup will produce both reports if they exist)*. What do you think?

Reply:
cool. updated